android/app/src/main/java/edu/berkeley/eecs/e_mission/ConfirmSectionActivity.java

```diff
@@ -55,6 +30,16 @@
 import com.google.android.gms.maps.model.Polyline;
 import com.google.android.gms.maps.model.PolylineOptions;
 
+import org.json.JSONArray;

```

Comment:
There are no substantive changes in this file except for the import cleanups, right? Nothing wrong with import cleanup, but it might be good to highlight that in the pull request message (i.e. Bonus fix: cleanup imports in ConfirmSectionActivity) to make it easier to review.